docs/tasks/access-kubernetes-api/setup-api-aggregator.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Setup the API Aggregator
+assignees:
+- chenopis
+---
+
+{% capture overview %}
+
+Setting up the API Aggregator will allow the Kubernetes apiserver to be extended with additional APIs, which are not part of the core Kubernetes APIs. 
```

Comment:
nit: "allows" instead of "will allow" (there are a couple unnecessary future tenses scattered around...might want to just Ctrl-f "will")